src/app/design/frontend/base/default/template/shopgate/cloudapi/pipelineRequest.phtml

```diff
@@ -1,6 +1,8 @@
 <meta id="libshopgate">
 <script type="text/javascript">
+    let shopBaseUrl = '<?php echo $this->getShopBaseUrl() ?>';
+    let isInApp = '<?php echo Mage::helper('shopgate_cloudapi/request')->isShopgateRequest(); ?>';
     document.observe("dom:loaded", function () {
```

Comment:
Please use const intead of let. Both variables never change.

Reply:
Done. They were const some time ago, but there was an issue within the plugin, which leads to double loading this phtml, so i temporary had to declare them by "let". Absolutely right to use const here, changed this.